napari/utils/misc.py

```diff
@@ -147,11 +147,11 @@ def is_iterable(arg, color=False, allow_none=False):
     provided and the argument is a 1-D array of length 3 or 4 then the input
     is taken to not be iterable. If allow_none is True, `None` is considered iterable.
     """
-    if (
-        (arg is None and not allow_none)
-        or isinstance(arg, str)
-        or np.isscalar(arg)
-    ):
+    if arg is None and not allow_none:
+        return False
+    if isinstance(arg, (str, Enum)):
+        return False
+    if np.isscalar(arg):
```

Comment:
Why split on multiple clauses?

Reply:
I thought it looked hard to parse when it was in a single clause. This is clearer because it's divided logically.